.github/workflows/integration-test.yml

```
@@ -3,7 +3,7 @@ on:
   workflow_call:
     inputs:
       runtime_tag:
-        description: "Tag for the functions runner image"
+        description: "Tag for the deno runner image"
```

this is used for both runtimes
```suggestion
        description: "Tag for the deno and python runner images"
```